BUILD_DETAIL.md

```diff
@@ -97,23 +97,22 @@ $ bundle exec yard doc --no-cache
 $ bin/yard doc --no-cache
 ```
 
-## RuboCop
+## Rubocop
 
-We use [RuboCop](https://github.com/rubocop-hq/rubocop) to enforce style
-conventions on the project so that the code has stylistic consistency
-throughout. Run with:
+We use [Rubocop](https://github.com/bbatsov/rubocop) to enforce style conventions on the project so
+that the code has stylistic consistency throughout. Run with:
```

Comment:
This was a regression surely?

Reply:
Seems so. I remember I was fixing this, but I wasn't fully aware that `BUILD_DETAIL.md` is coming from `rspec-dev`.

Reply:
Prs opened from rspec-dev